mongoose.c

```diff
@@ -5717,17 +5727,8 @@ MG_IRAM bool mg_imxrt_swap(void) {
   return true;
 }
 
-static inline void spin(volatile uint32_t count) {
-  while (count--) (void) 0;
-}
-
-static inline void flash_wait(void) {
-  while ((*((volatile uint32_t *) (0x402A8000 + 0xE0)) & MG_BIT(1)) == 0)
-    spin(1);
-}
-
 MG_IRAM static bool mg_imxrt_write(void *addr, const void *buf, size_t len) {
-  struct mg_flexspi_nor_config config;
+  struct mg_flexspi_nor_config *config;
```

Comment:
This works for RT1020, but will fail for RT1060, because there we need to have storage for the ROM function to fill, that's why I declared the struct and a pointer to it.
The struct is allocated in the stack;
for the RT1020, it is not used and freed at the end of the function call
for the RT1060, it is filled by the ROM call and freed at the end of the function call.

Reply:
Ah, you are right. My bad. Let me change that.